k8s/prod/common/ccd/data-store-api.yaml

```diff
@@ -19,18 +19,6 @@ spec:
   values:
     java:
       replicas: 9
-      keyVaults:
-        ccd:
-          secrets:
-          - data-store-api-POSTGRES-USER-V11
```

Comment:
remove DATA_STORE_DB_USERNAME: ccd@ccd-data-store-api-postgres-db-v11-prod entry as bootstrap has that already.